src/dali/plugin/input/rest/kucoin_futures.py

```diff
@@ -279,8 +275,6 @@ def _process_implicit_api(
     ) -> None:
         in_transactions.clear()
```

Comment:
It seems a bit dangerous to clear these. If you or someone else later implements deposits and withdrawals, these `clear()`s will clear out those transactions. I would take them out unless you have a good reason for them.

Reply:
fixed